runtime/compiler/env/VMJ9.cpp

```diff
@@ -3112,6 +3112,18 @@ TR_J9VMBase::isMethodTracingEnabled(TR_OpaqueMethodBlock *method)
    return VM_VMHelpers::methodBeingTraced(_jitConfig->javaVM, (J9Method *)method);
    }
 
+bool
+TR_J9VMBase::isLambdaFormGeneratedMethod(TR_OpaqueMethodBlock *method)
+   {
+   return VM_VMHelpers::isLambdaFormGeneratedMethod(vmThread(), (J9Method *)method);
+   }
+
```

Comment:
I don't remember seeing this last time. This will require an implementation for JITServer. @dmitry-ten

Reply:
I don't see this method used anywhere in this PR.
Where is it supposed to be called from?

Reply:
It's called from MethodHandleTransformer.cpp

Reply:
Ok, I see. Yeah, in that case I'll have to override it for JITServer as well.